fastai/data/core.py

```diff
@@ -217,8 +217,20 @@ def n_subsets(self): return len(self.splits)
     def _new(self, items, **kwargs): return super()._new(items, splits=self.splits, **kwargs)
     def subset(self): raise NotImplemented
 
-    def dataloaders(self, bs=64, shuffle_train=None, shuffle=True, val_shuffle=False,n=None, path='.', dl_type=None, dl_kwargs=None,
-                    device=None,drop_last=None,val_bs=None, **kwargs):
+    def dataloaders(self,
+        bs:int=64, # Size of batch
+        shuffle_train:bool=None, # (Deprecated, use `shuffle`) Whether to shuffle training `DataLoader`
+        shuffle:bool=True, # Whether to shuffle training `DataLoader`
+        val_shuffle:bool=False, # Whether to shuffle validation `DataLoader`
+        n:int=None, # Size of `Datasets` used to create `DataLoader`
+        path:(str, Path)='.', # Path to put in `DataLoaders`
+        dl_type:TfmdDL=None, # Type of `DataLoader`
+        dl_kwargs:list=None, # list of kwargs to pass to individual `DataLoader`s
+        device:torch.device=None, # Device to put `DataLoaders`
+        drop_last:bool=None, # Whether to drop last incomplete batch
+        val_bs:int=None, # Size of batch for validation `DataLoader`
+        **kwargs
+    ) -> DataLoaders:
```

Comment:
Some suggestions to improve conciseness and add info about default setting. Normally, I'd say to remove the "List of" but in `dl_kwargs` case I think could be useful to keep.

```suggestion
    def dataloaders(self,
        bs:int=64, # Batch size
        shuffle_train:bool=None, # (Deprecated, use `shuffle`) Whether to shuffle training `DataLoader`
        shuffle:bool=True, # Shuffle training `DataLoader`
        val_shuffle:bool=False, # Shuffle validation `DataLoader`
        n:int=None, # Size of `Datasets` used to create `DataLoader`
        path:(str, Path)='.', # Path to put in `DataLoaders`
        dl_type:TfmdDL=None, # Type of `DataLoader`
        dl_kwargs:list=None, # List of kwargs to pass to individual `DataLoader`s
        device:torch.device=None, # Device to put `DataLoaders`
        drop_last:bool=None, # Drop last incomplete batch, defaults to `shuffle`
        val_bs:int=None, # Validation batch size, defaults to `bs`
        **kwargs
    ) -> DataLoaders:
```